### PR TITLE
Added map to display the list of upcoming events.

### PR DESCRIPTION
**Pull request by hhamon**

### CODE REVIEW

File: front/pages/events_map.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*
+ * Upcoming events map
+ */
+export default (mapFactory, api) => {
+    api.getUpcomingEvents((events) => {
+        const map = mapFactory.createMap(dom('#map'), {
+            center: { lat: 46.7699, lng: 2.4279 },
```
Comment (mykiwi) [inline front/pages/events_map.js]: maybe you should explain what are these coordinates (I guess it is to display the whole France?)

File: tests/AppBundle/Controller/PageControllerTest.php [hunk 2]
```diff
@@ -16,18 +18,30 @@ class PageControllerTest extends SqliteWebTestCase
...
      */
-    public function testPages($path)
+    public function testPages(string $path)
     {
         $this->client->request(Request::METHOD_GET, $path);
         $this->assertResponseStatusCode(Response::HTTP_OK, $response = $this->client->getResponse());
     }
 
-    public function testMap()
+    public function testCommitteesMap()
```
Comment (mykiwi) [inline tests/AppBundle/Controller/PageControllerTest.php]: `@group functionnal` ?

File: tests/AppBundle/Controller/PageControllerTest.php [hunk 3]
```diff
@@ -16,18 +18,30 @@ class PageControllerTest extends SqliteWebTestCase
...
         $crawler = $this->client->request(Request::METHOD_GET, '/le-mouvement/la-carte');
 
         $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
         $this->assertSame(1, $crawler->filter('html:contains("La carte En Marche !")')->count());
+        $this->assertContains('10 adhérents', $crawler->filter('.committees-map__counter')->first()->text());
+        $this->assertContains('6 comités', $crawler->filter('.committees-map__counter')->eq(1)->text());
+        $this->assertContains('6 événements', $crawler->filter('.committees-map__counter')->last()->text());
+    }
+
+    public function testCommitteesEventsMap()
```
Comment (mykiwi) [inline tests/AppBundle/Controller/PageControllerTest.php]: `@group functionnal` ?

File: front/pages/events_map.js (hunk 1 above)
Comment (hhamon) [inline front/pages/events_map.js]: I guess so! I just copied/pasted them from the JavaScript code to display the map of committees.

File: tests/AppBundle/Controller/PageControllerTest.php (hunk 2 above)
Comment (hhamon) [inline tests/AppBundle/Controller/PageControllerTest.php]: yes, good catch! I'll put it on top of the class.

File: front/pages/events_map.js (hunk 1 above)
Comment (hhamon) [inline front/pages/events_map.js]: Fixed.

File: tests/AppBundle/Controller/PageControllerTest.php (hunk 2 above)
Comment (hhamon) [inline tests/AppBundle/Controller/PageControllerTest.php]: Fixed.

File: tests/AppBundle/Controller/PageControllerTest.php (hunk 3 above)
Comment (hhamon) [inline tests/AppBundle/Controller/PageControllerTest.php]: Fixed.

**Pull request merged**